tailwind.config.js

```diff
@@ -1,79 +1,130 @@
 module.exports = {
-  purge: ["./pages/**/*.{js,ts,jsx,tsx}", "./components/**/*.{js,ts,jsx,tsx}"],
+  purge: {
+    content: [
+      "./pages/**/*.{js,ts,jsx,tsx}",
+      "./components/**/*.{js,ts,jsx,tsx}",
+    ],
+    options: {
+      safelist: [
+        "border-custom-blue-experiment-blue",
+        "border-gray-experiment",
+        "bg-custom-blue-experiment-blue",
+        "bg-gray-experiment",
+        "opacity-40",
+        "opacity-60",
+        "opacity-80",
+        "opacity-100",
+      ],
+    },
+  },
   darkMode: false, // or 'media' or 'class'
   theme: {
     fontFamily: {
       display: ["Lato"],
       body: ["Noto sans"],
     },
     fontSize: {
-      xxs: ["13px", "16px"],
+      xxs: ["12px", "16px"],
       xs: ["14px", "16px"],
       sm: ["16px", "22px"],
       base: ["18px", "28px"],
       p: ["20px", "30px"],
-      h6: ["19px", "28.5px"],
-      h5: ["20px", "30px"],
```

Comment:
Getting rid of this results in the `Find information about` text in our topic boxes being too big. Needs to have this sizing as per designs.

Reply:
uh!. those two lines aren't on the alpha site, configuration.  re added